src/BlockInfo.h

```diff
@@ -40,9 +40,6 @@ class cBlockInfo
 	/** Does a block disperse sky light? (only relevant for transparent blocks) */
 	bool m_IsSkylightDispersant;
 
-	/** Can this block hold snow atop? */
-	bool m_IsSnowable;
```

Comment:
This breaks the lua api, you should manually bind accessors for `m_IsSnowable` like when `m_PlaceSound` was removed in #3897.